### PR TITLE
created the section for Service Meshes you can learn

### LINKED ISSUE #1896: [Learn] Create a Service Meshes you can learn section
**Current State:**
The Learn Layer5 website is missing the  "Service Meshes you can learn " section

**Desired State:**
Create this sections on the Learn Layer5 site

Screenshot 2021-06-01 at 09 58 54

---
**Contributor Resources**
The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- Check the Learn Layer5 site [Design Spec](https://docs.google.com/document/d/1tQ9tLbetBVKP7z76FR19vsxbAfoFHZzZYxCt7lJpAhU/edit) to know more about the site structure and development plans.
- Checkout the Learn Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/?node-id=4853%3A0).

### PR DESCRIPTION
**Pull request by Anita-ihuman**
Signed-off-by: Anita-ihuman <charlesanita403@gmail.com>

**Description**
Created a section for the Service Meshes you can learn.
Screenshot 2021-06-05 at 21 37 50

This PR fixes #1896

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit fd32aba06cb208042337d11749b31564de060c90 at: https://60bbe2fc1767515d0ffd6539--layer5ng.netlify.app

**Comment by chinmaym07** [comment]
This looks good @Anita-ihuman ..Thankyou 👍🏻 🚀 🥇 
Some refactoring required here as we will need to get the data of these `Service Meshes you can learn` from each course mdx file itself ..
Probably you can create array of objects which has properties named `imagepath` & `name` for the type of service mesh in the course mdx file.Then specify that same array attribute name in the graphql query to get the array ..then render this structure using the data that you get ..

**Comment by Anita-ihuman** [comment]
Alright, will try this and get back to you.

**Comment by Nikhil-Ladha** [comment]
Looks good @Anita-ihuman . No pressure on dynamic data, we are good with static data as well for now. But, if you are comfortable with it, you can give it a shot, it will be a bonus to the PR :)

**Comment by Anita-ihuman** [comment]
Alright, @Nikhil-Ladha  I will git it a shot. Before giving up.

**Comment by l5io** [comment]
🚀 Preview for commit b5a2410e72d744512287e7709235161cdab533ea at: https://60c00334270378008fd1a2bd--layer5ng.netlify.app

**Comment by Nikhil-Ladha** [comment]
Hey, @Anita-ihuman that's not how we meant to achieve dynamic data, it was more a graphql way to fetch the meshes from the MDX file and so. Anyway, you can revert the last commits, and only keep the static data for now. The dynamic part can be handled later. Once, you revert it do signal here so that I will review and merge the PR.

**Comment by Anita-ihuman** [comment]
@Nikhil-Ladha I just reverted it

**Comment by l5io** [comment]
🚀 Preview for commit 6ea5fb7c99c79dd9bd234c6379e627a50207992b at: https://60c3981560361f0f0da994d9--layer5ng.netlify.app

**Review by Jashpatel1 — APPROVED** [review]
Looks good to me.

File: src/components/Learn-Components/Course-Overview-page/index.js [hunk 1]
```diff
@@ -1,24 +1,30 @@
...
+import Course1 from "../../../assets/images/learn/sercice-mesh-course1.png";
 
 import { Link } from "gatsby";
-import { IoDocumentTextOutline, IoVideocam,IoChevronBackOutline } from "react-icons/io5";
+
+import {
+  IoDocumentTextOutline,
+  IoVideocam,
+  IoChevronBackOutline,
+} from "react-icons/io5";
```
Comment (Nikhil-Ladha) [inline src/components/Learn-Components/Course-Overview-page/index.js:13]: These icons are not used anywhere, it can be removed.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Remove the unused images which are being added in the PR.

File: src/components/Learn-Components/Course-Overview-page/index.js (hunk 1 above)
Comment (Jashpatel1) [reply src/components/Learn-Components/Course-Overview-page/index.js:13]: They are used below I guess.

Comment (Nikhil-Ladha) [reply src/components/Learn-Components/Course-Overview-page/index.js:13]: Ahh...right. But, do remove the unused images @Anita-ihuman .

**Comment by l5io** [comment]
🚀 Preview for commit afce09f936b77020e9253deaa9860a43322e6531 at: https://60c74a9aebfb7190e0046b66--layer5ng.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]
Thank you @Anita-ihuman!

**Pull request merged**